### DISCUSSION TITLE
ELEGOO Saturn calibration

### DISCUSSION THREAD

**Discussion by LordRofl** [sn4k3/UVtools#225] [Q&A] [answered]
Hello i just tried the resin calibration test and it sticks to just the initial exposure time.

Can anyone confirm they run a successful exposure test with the Saturn?

**Comment by sn4k3** [comment, marked as answer]
You need the latest firmware.
Which version do you have?

**Reply by LordRofl** [reply]
4.4.1

**Reply by sn4k3** [reply]
ok thats compatible. Send screenshot of your Exposure table from UVtools calibration dialog or send the generated ctb

**Reply by LordRofl** [reply]
Did another firmware ugrade to 15.26 (PJ V5) and redo the exposure calibration file generation with uvtools and now it works!
